File: Data Structures & Algorithms/number-of-senior-citizens/submission-0.cpp
class Solution {
public:
    int countSeniors(vector<string>& details) {
        int ans = 0;
        for(const auto &st: details)
            if((st[11] - '0') * 10 + (st[12] - '0') > 60)
                ans++;
        return ans;
    }
};

File: Data Structures & Algorithms/max-consecutive-ones/submission-0.py
class Solution:
    def findMaxConsecutiveOnes(self, nums: List[int]) -> int:
        ans, ones = 0, 0
        for n in nums:
            if n:
                ones += 1
            else:
                ans = max(ans, ones)
                ones = 0
        return max(ans, ones)